Source/Core/Common/Arm64Emitter.cpp

```diff
@@ -38,7 +38,8 @@ static uint64_t LargestPowerOf2Divisor(uint64_t value)
   return value & -(int64_t)value;
 }
 
-bool IsImmArithmetic(uint64_t input, u32* val, bool* shift)
+// For ADD/SUB
+static bool IsImmArithmetic(uint64_t input, u32* val, bool* shift)
```

Comment:
Why not anonymous namespace instead of static?

Reply:
Not sure why this shows as outdated :/

Reply:
Will amend it to use an anonymous namespace in a few minutes